rados/README.md

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright (c) 2014 - 2015 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on RADOS of Ceph.
+
+### 1. Start RADOS
+
+After you start your Ceph cluster, check your cluster’s health first. You can check on the health of your cluster with the following:
+
+    ceph health
+
+### 2. Install Java and Maven
+
+### 3. Set Up YCSB
+
+Git clone YCSB and compile:
+
+    git clone -b dev-ceph-rados https://github.com/jaemyoun/YCSB.git
```

Comment:
should probably just tell folks to download. Definitely should refer to the correct git repo if it stays as a clone.